WordPress/gutenberg
# Switch background color to text color on block separator

## What?

Fixes #43660 using the first option: 
 - When rendering theme.json, add a CSS property with color or border-color and the defined color.

## Why?

Defining the separator color in a style variation, using background on theme.json is not working. Most variations are done with text color or border color.

## How?

Tweaking the stylesheet creation to add the text color if needed. Skipping if there is already a color set.

## Testing Instructions




- Create a style variation of a block theme
- Change the color for the separator block through style-variant.json, using text as the field.
- Check the website and see that the color has changed.
